src/components/Profile/Profile.styles.ts

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+
+export const ProfileContainer = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 20rem;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+`;
+
+export const ProfileImage = styled.img`
+  border-radius: 50%;
+  width: 100px;
+  height: 100px;
+`;
+
+export const ProfileContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: flex-start;
+  padding: 0 20px;
+`;
+
+export const ProfileName = styled.h1`
+  margin: 0;
+  font-size: 20px;
+  font-weight: bold;
+`;
```

Comment:
To improve accessibility it's best if we don't use the tag h1 here, that is because screen headers use title tags to jump between sections inside the screen.

```suggestion
export const Name = styled.span`
  font-size: 1.2rem;
  font-weight: bold;
`;
```